static/js/popovers.js

```diff
@@ -870,11 +882,19 @@ exports.register_click_handlers = function () {
     });
     */
 
+    $("body").on("click", ".status_info", (e) => {
+        e.stopPropagation();
+        let option;
```

Comment:
I think you need to define this variable otherwise, it will always be undefined and never this will always perform a `server_revoke_away` action.

Reply:
which variable , and where do you mean  I should define it

Reply:
Oh sorry, I wrote it wrong, English is not my native tongue. What I mean is that I think you need to attribute the value oof `option`